### PR TITLE
fix: predict spinner cannot be read

### PR DESCRIPTION
**Pull request by tianxiangs**

### CODE REVIEW

File: src/react/components/pages/predict/predictPage.tsx [hunk 1]
```diff
@@ -196,7 +203,7 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
                             {!this.state.predictionLoaded &&
                                 <div className="loading-container">
                                     <Spinner
-                                        role="alert"
```
Comment (stew-ro) [inline src/react/components/pages/predict/predictPage.tsx]: I think you just need to set the ariaLive attribute (Politeness setting for label update announcement) of the spinner to assertive. By default it is polite. https://developer.microsoft.com/en-us/fabric#/controls/web/spinner

Comment (tianxiangs) [reply src/react/components/pages/predict/predictPage.tsx]: I tried, it does not work

Comment (stew-ro) [reply src/react/components/pages/predict/predictPage.tsx]: With both ariaLabel and ariaLive?

File: src/react/components/pages/predict/predictPage.tsx.orig [hunk 2]
```diff
@@ -0,0 +1,673 @@
+// Copyright (c) Microsoft Corporation.
```
Comment (kunzms) [inline src/react/components/pages/predict/predictPage.tsx.orig]: remove this file

Comment (tianxiangs) [reply src/react/components/pages/predict/predictPage.tsx.orig]: Already removed, maybe refresh

**Pull request merged**